README.md

```diff
@@ -64,7 +64,7 @@ EXAMPLES
   $ jamsocket login
 ```
 
-_See code: [dist/commands/login.ts](https://github.com/drifting-in-space/jamsocket-cli/blob/v0.1.1/dist/commands/login.ts)_
+_See code: [dist/commands/login.ts](https://github.com/drifting-in-space/jamsocket-cli/blob/v0.1.2/dist/commands/login.ts)_
```

Comment:
Both the new URL and the old one 404 for me

Reply:
Ahh, the issue with the URLs looks like it's always existed. Looks like these links that oclif generates don't work if you don't check in `dist`. Yet, the `.gitignore` that oclif starts you off with has `dist` in it. And even if you check in `dist`, the links still won't work since `dist` doesn't include `.ts` files  🤔

Reply:
Tried looking through their Issues and PRs, not finding anything relevant. :-( This is the bug: https://github.com/oclif/oclif/blob/main/src/commands/readme.ts#L216 - they are splitting on the default topic separator (`:`) instead of the custom one we're providing in the packagejson. (We use a space as our separator so our commands look like `jamsocket service create` instead of `jamsocket service:create`.) I could open a PR, but it looks like there are lots of spots where the `:` is hardcoded, not sure if changing all of them will address the issue. Also, it looks like my other PR from a few weeks ago hasn't been looked at yet either: https://github.com/oclif/core/pull/414 :-(

Reply:
UPDATE: got a fix here - didn't need to fork after all, thank god: https://github.com/drifting-in-space/jamsocket-cli/pull/22